tests/espressif/esp32_devkitc_esp_wrover_kit/common/application_code/espressif_code/amazon-freertos-tests/component.mk

```diff
@@ -65,7 +65,9 @@ COMPONENT_OBJEXCLUDE := $(AMAZON_FREERTOS_LIB_DIR)/FreeRTOS-Plus-TCP/source/port
         $(AMAZON_FREERTOS_LIB_DIR)/FreeRTOS-Plus-POSIX/source/FreeRTOS_POSIX_pthread_cond.o \
         $(AMAZON_FREERTOS_LIB_DIR)/FreeRTOS-Plus-POSIX/source/FreeRTOS_POSIX_sched.o \
         $(AMAZON_FREERTOS_TESTS_DIR)/common/ota/aws_test_ota_cbor.o \
-	$(AMAZON_FREERTOS_LIB_DIR)/lob/ota/aws_ota_agent.o
+	$(AMAZON_FREERTOS_LIB_DIR)/ota/aws_ota_agent.o \
+        $(AMAZON_FREERTOS_LIB_DIR)/utils/platform/iot_clock_posix.o \
```

Comment:
If we do completely remove the dependency on posix for this we can remove the posix implementations as well as this exclusion. I believe we need the posix implementations in the C-SDK for people not running on FreeRTOS, but this is aFR so here having the option does not really make sense?

Reply:
Agreed.  Remove the POSIX implementations.